apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/api/CacheKeyResolver.kt

```diff
@@ -40,7 +41,12 @@ abstract class CacheKeyResolver : CacheResolver {
    */
   open fun listOfCacheKeysForField(field: CompiledField, variables: Executable.Variables): List<CacheKey?>? = null
 
-  final override fun resolveField(field: CompiledField, variables: Executable.Variables, parent: Map<String, Any?>, parentId: String): Any? {
+  final override fun resolveField(
+      field: CompiledField,
+      variables: Executable.Variables,
+      parent: Map<String, @JvmSuppressWildcards Any?>,
```

Comment:
Generics are hard 😅 . I trust you on this one :)

Reply:
It just didn't work from Java without this - I noticed it was there on the super class, and that fixed it 😄